data/serializer/src/main/java/tech/pegasys/teku/api/response/GetForkResponse.java

```diff
@@ -50,6 +50,14 @@ public GetForkResponse(
     this.genesis_validators_root = genesis_validators_root;
   }
 
+  public GetForkResponse(
+      final tech.pegasys.teku.api.schema.Fork fork, final Bytes32 genesis_validators_root) {
+    this.previous_version = fork.previous_version;
+    this.current_version = fork.current_version;
+    this.epoch = fork.epoch;
+    this.genesis_validators_root = genesis_validators_root;
+  }
```

Comment:
Did we actually wind up using this?

Reply:
nope! I probably could have, but i created by hand. I'll clean it up.